backend/internal/controller/server/msg_handler.go

```diff
@@ -67,6 +67,9 @@ func (s server) createResponse(reader io.Reader) ([]byte, error) {
 	case adapter.MsgGoal:
 		err := s.game.AddGoal(message.Team)
 		return nil, err
+	case adapter.MsgPosition:
+		log.Trace("X: ", message.X, " Y: ", message.Y)
```

Comment:
Why did you decide to use the `trace` level? Maybe better would be to use `debug`? 

I would suggest improving the message because printing info in format like `X:..., Y:...` is a great way to make log hell in your app. A little context would be expected

Reply:
We use `trace` because this message is only to check if we have received the coordinates of the ball. At the `debug` level this would generate a lot of unnecessary messages covering up other, more important messages.

We will also improve the `X:..., Y:...` format to give more context.